app/templates/add.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% block content %}
+<form method="post" name="add">
+    {{form.hidden_tag()}}
+    <p>
+        Please enter book name:<br>
+        {{form.book_name(size=100)}}<br>

```

Comment:
It is better to use some macro. I think that column definitions in the form already configure field sizes like string length etc. The visual styling has to be done with CSS. In the next issue you could use Bootstrap CSS framework and use some bootstrap macro for the forms.